### ISSUE TITLE
Whitespace characters are near invisible in High Contrast theme

### ISSUE THREAD

**Issue by shweaver-MSFT** [microsoft/vscode#124612]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.56.2
- OS Version: 21H1 build 19043.985

Steps to Reproduce:

1. Set Theme to High Contrast
2. Via Settings, enable All whitespace characters
3. Observe some code

![image](https://user-images.githubusercontent.com/25832310/119548601-3be51d80-bd4b-11eb-9343-714b418b4564.png)

The whitespace characters are there, but very difficult to distinguish. I am a full time High Contrast theme user and the lack of contrast for whitespace characters forces me to switch themes if I want to actually see them.

**Comment by misolori** [comment]
Updated the colors so these are more prominent:

![image](https://user-images.githubusercontent.com/35271042/120039823-5ffa6600-bfba-11eb-9642-233493e68821.png)

Which also allows them to be visible on selection:

![image](https://user-images.githubusercontent.com/35271042/120039859-71dc0900-bfba-11eb-86c0-d32105b8d991.png)

**Comment by andreamah** [comment]
I noticed that the whitespace is not rendering upon selection if I select `selection` on the "Render Whitespace" option.
![image](https://user-images.githubusercontent.com/31675041/120701803-8d3e8c80-c470-11eb-8647-f0b63a743ced.png)

![image](https://user-images.githubusercontent.com/31675041/120701914-b19a6900-c470-11eb-98bf-937cc34f351c.png)

It seems to just not be working on the high contrast theme.

**Comment by misolori** [comment]
It looks like the `selection` has been broken for a while now sine I can repro in stable. The fix here is to make this appear better for `boundary` and `all`. I'll create a separate issue for the selection state.
